Update 2019-05-01-may-update.md

leannethelibrarian opened the pull request:
Added the update text (with some minor changes from version in the google folder, to reflect context on this website)

leannethelibrarian committed:
Update 2019-05-01-may-update.md

Added the update text (with some minor changes from version in the google folder, to reflect context on this website)

mediagestalt reviewed (commented):
Is the date on this actually from May 2019?

leannethelibrarian commented:
No...it was supposed to be September 2019. My bad. Now to figure out how to change it...

leannethelibrarian committed:
Update and rename 2019-05-01-may-update.md to 2019-09-01-sept-update.md

More edits - fixed issue introduced with colon in post title, plus fixed the date of the post.
